cpp/include/cudf/quantiles.hpp

```diff
@@ -42,5 +42,36 @@ quantile(column_view const& input,
          interpolation interp = interpolation::LINEAR,
          order_info column_order = {});
 
+/* @brief Gathers the rows corrosponding to the desired quantiles.
+ *
+ * The indices used to gather rows are computed by interpolating between the
+ * index on either side of the desired quantile. Since some columns may be
+ * non-arithmetic, interpolation between rows is limited to non-arithmetic
+ * strategies.
+ *
+ * quantiles `<= 0` corrospond to row `0`. (first)
```

Comment:
There are some good spell checkers for IDEs.

```suggestion
 * quantiles `<= 0` correspond to row `0`. (first)
```

Reply:
I've installed Code Spell Checker for VS Code after making sure it's an offline-only tool.

https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker